backend/src/dataclasses/playlist_info.py

```diff
@@ -1,26 +1,38 @@
-from dataclasses import dataclass
-from typing import List
+from pydantic import BaseModel
+from typing import List, Optional
 
-from src.dataclasses.album import Album
 from src.dataclasses.image import Image
 from src.dataclasses.track import Track
 
 
-@dataclass
-class PlaylistTrack:
+class PlaylistTrack(BaseModel):
     added_at: str
     track: Track
 
-@dataclass
-class PlaylistTracksContainer:
+
+class PlaylistTracksContainer(BaseModel):
     total: int
     items: List[PlaylistTrack]
 
 
-@dataclass
-class PlaylistInfo:
+class PlaylistInfoTracks(BaseModel):
+    total: int
+    href: str
+
+
+class SimplifiedPlaylist(BaseModel):
     id: str
     name: str
     description: str
     images: List[Image]
```

Comment:
I like the clarity of Pydantic errors! It turns out that I have a couple of old playlists that no longer have a single valid Spotify song on them which means they turn up without any valid images!😂 
```suggestion
    images: Optional[List[Image]]
```

Reply:
Haha yep, caught that on the next ticket as you spotted. Clearly just listening to too much indie music Jack!